### DISCUSSION TITLE
Front-end Update (+ Checkbox) 동작 불능

### DISCUSSION THREAD

**Discussion by 0-ki** [fsoftwareengineer/todo-application#104] [질문과 답변] [answered]
https://github.com/0-ki/todoApp

1. Todo.js의 <Checkbox> 태그에서, checked가 동작하지 않고, Checked 와 같이 mui 문법과 다르게 할 경우는 동작을 합니다.
2. editEventHandler 함수가 

    const editEventHandler = (e) => {
        const thisItem = item;
        thisItem.title = e.target.value;
        setItem(thisItem);
    };

와 같이 구현되어 있는데, 타이핑 시 한 개씩 늦게 밀려서 들어가게 되고, 마지막 입력값만 추가됩니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
확인 후 코드를 업데이트하도록 하겠습니다.
